.github/workflows/deploy.yml

```
@@ -0,0 +1,583 @@
+            runner: ubuntu-24.04-arm
+            platform_filter: linux-arm64
+          - target: x86_64-pc-windows-msvc
+            runner: windows-latest
+            platform_filter: windows-x64
+          - target: aarch64-apple-darwin
+            runner: macos-latest
+            platform_filter: macos-arm64
+          - target: x86_64-apple-darwin
+            runner: macos-latest
+            platform_filter: macos-x64
+    if: contains(needs.validate.outputs.platforms, matrix.platform_filter)
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)